app/config/packages/routing.yaml

```
@@ -4,4 +4,4 @@ framework:
 
         # Configure how to generate URLs in non-HTTP contexts, such as CLI commands.
         # See https://symfony.com/doc/current/routing.html#generating-urls-in-commands
-        #default_uri: http://localhost
+        default_uri: '%env(resolve:DOMAIN_SCHEME)%://%env(resolve:DOMAIN)%'
```

Je suggère de remplacer par une variable unique :

```suggestion
        default_uri: '%env(APP_URL)%'
```

L’intérêt c’est que ça pourra fonctionner aussi s'il y a un port dans l’URL de l’application.